src/live-preview.ts

```diff
@@ -178,6 +197,43 @@ export default class LivePreview {
         }
     }
 
+    private removeEditButtonStyle(e: MouseEvent) {
+        if (!this.currentElementBesideTooltip || !this.tooltip) return false;
```

Comment:
The size of this file is increasing. Let's consider moving these functions into new files.

Reply:
sure, will move this function to utils files